core/io/file_access_zip.cpp

```diff
@@ -86,9 +86,7 @@ static long godot_seek(voidpf opaque, voidpf stream, uLong offset, int origin) {
 static int godot_close(voidpf opaque, voidpf stream) {
 	FileAccess *f = (FileAccess *)stream;
 	if (f) {
-		f->close();
 		memdelete(f);
-		f = nullptr;
```

Comment:
It seemed to me like this isn't necessary since `f` is a local variable and it's not used after this. But might need someone else to confirm as it's pretty core.